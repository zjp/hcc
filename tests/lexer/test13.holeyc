'\a 
bool false;
bool true;
\ 
\b
\c
defg
