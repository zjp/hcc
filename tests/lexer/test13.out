BOOL [2,1]
FALSE [2,6]
SEMICOLON [2,11]
BOOL [3,1]
TRUE [3,6]
SEMICOLON [3,10]
ID:b [5,2]
ID:c [6,2]
ID:defg [7,1]
EOF [8,1]
